source/change-streams.rst

```diff
@@ -320,8 +365,60 @@ Presently change streams support only a subset of available aggregation stages:
 
 A driver MUST NOT throw an exception if any unsupported stage is provided, but instead depend on the server to return an error.
 
-The aggregate helper method MUST have no new logic related to the ``$changeStream`` stage. Drivers MUST be capable of handling `TAILABLE_AWAIT <https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst#read>`_  cursors from the aggregate command in the same way they handle such cursors from find.
+The aggregate helper methods MUST have no new logic related to the ``$changeStream`` stage. Drivers MUST be capable of handling `TAILABLE_AWAIT <https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst#read>`_  cursors from the aggregate command in the same way they handle such cursors from find.
+
```

Comment:
I think it's worth noting that the reason these command definitions did not originally exist in our spec is because they are (will be) documented by the server documentation. Let's consider removing them again

Reply:
Having them in the spec is consistent with other CRUD-release driver specs.  It specifies how to translate from the driver API to the server API.